### DISCUSSION TITLE
Backup Tables

### DISCUSSION THREAD

**Discussion by WatheqAlshowaiter** [HassanZahirnia/laravel-package-ocean#237] [Package Suggestions]
Backup single or multiple database tables with ease.

By adding `BackupTables::generateBackup('users')` and that is it. You can also backup multiple tables or simple pass models instead of tables if you want.

This is different from the Spatie backup package because it is focused on tables for certain cases, not the regular backups for all applications like what the Spatie package did.

package link with examples and features in detail: 
https://github.com/WatheqAlshowaiter/backup-tables

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
